src/OutputWriters/windowed_time_average.jl

```diff
@@ -102,7 +115,9 @@ function run_diagnostic(model, wta::WindowedTimeAverage)
         wta.window_start_iteration = model.clock.iteration
         wta.previous_collection_time = model.clock.time
 
-    elseif model.clock.time - wta.window_start_time >= wta.time_window 
+    elseif model.clock.time >= wta.previous_interval_stop_time + wta.time_interval # output is imminent
+    #elseif model.clock.time - wta.window_start_time >= wta.time_window 
```

Comment:
Any reason to leave this commented out?

Reply:
No, that is a mistake! I'll delete it. After I add a test.